src/compute/src/server.rs

```diff
@@ -166,22 +166,19 @@ pub async fn compute_node_serve(
 
     let mut sub_tasks: Vec<(JoinHandle<()>, Sender<()>)> = vec![];
     // Initialize the metrics subsystem.
-
-    monitor_process(&registry).unwrap();
-    let source_metrics = Arc::new(SourceMetrics::new(registry.clone()));
-    let hummock_metrics = Arc::new(HummockMetrics::new(registry.clone()));
-    let streaming_metrics = Arc::new(StreamingMetrics::new(registry.clone()));
-    let batch_task_metrics = Arc::new(BatchTaskMetrics::new(registry.clone()));
-    let batch_executor_metrics = Arc::new(BatchExecutorMetrics::new(registry.clone()));
-    let batch_manager_metrics = BatchManagerMetrics::new(registry.clone());
-    let exchange_srv_metrics = Arc::new(ExchangeServiceMetrics::new(registry.clone()));
+    let source_metrics = Arc::new(GLOBAL_SOURCE_METRICS.clone());
```

Comment:
Do we still need these? Is this for minimizing the number of changed lines?

It seems a little bit inconsistent while some are accessing `GLOBAL_SOURCE_METRICS` directly while the others are still using `source_metrics` passed from the caller.

Reply:
Yes. It's for minimizing changed lines. It's totally ok not to initialize it here but directly report to the global one in where reporting a metrics. 

We can create an issue to remove the code that pass the metrics all the way down to simplify our code.